indra/assemblers/indranet/assembler.py

```diff
@@ -236,13 +242,23 @@ def make_df(self, exclude_stmts=None, complex_members=3):
                 agA_ns, agA_id = get_ag_ns_id(agA)
```

Comment:
On lines 239-240 above, anything that is not an Influence, Association, Complex or Conversion is given a sign `None`. Presumably there should also be an if block handling Activation/Inhibition and Increase/DecreaseAmount to assign correct polarity. We should also add appropriate tests for these statements and others for which sign should properly be None (e.g. Modifications).

Reply:
Given that the existing sif dump pipeline (that doesn't use the IndraNet) doesn't include sign information, changing this is probably out of the scope of this PR and we can address later.